Site/SiteMultipartMailMessage.php

```diff
@@ -337,6 +339,34 @@ public function addBcc($email)
 	}
 
 	// }}}
+	// {{{ public function getCcArray()
+
+	/**
+	 * Gets an array of email addresses for CC
+	 *
+	 * @return arrays $email addresses for CC.
+	 */
+	public function getCcArray()
+	{
+		// array_filter will remove blank values
+		return array_filter($this->cc_list);
```

Comment:
This method doesn't work. It will still return blank entires that are in the array since `" " != false`.